Dockerfile-node

```diff
@@ -0,0 +1,12 @@
+FROM node:10
+
+COPY . /app
+
+WORKDIR /app
+
+RUN rm -rf node_modules
+
+RUN yarn install
+RUN yarn build
```

Comment:
These lines can be collapsed to a single run. In this way, we skip the creation of 3 layers of the container and we save a lot of space (: